tutorials/otel.md

```diff
@@ -0,0 +1,265 @@
+      - command:
+          - "/otelcol"
+          - "--config=/conf/otel-collector-config.yaml"
+#           Memory Ballast size should be max 1/3 to 1/2 of memory.
+          - "--mem-ballast-size-mib=683"
+        image: otel/opentelemetry-collector-dev:latest
+        name: otel-collector
+        resources:
+          limits:
+            cpu: 1
+            memory: 2Gi
+          requests:
+            cpu: 500m
+            memory: 1Gi
+        ports:
+        - containerPort: 55679 # Default endpoint for ZPages.
+        - containerPort: 4317 # Default endpoint for OpenTelemetry receiver.
+        - containerPort: 14250 # Default endpoint for Jaeger HTTP receiver.
+        - containerPort: 14268 # Default endpoint for Jaeger HTTP receiver.
+        - containerPort: 9411 # Default endpoint for Zipkin receiver.
+        - containerPort: 8888  # Default endpoint for querying metrics.
+        env:
+        - name: JAEGER_AGENT_HOST
+          valueFrom:
+            fieldRef:
+              fieldPath: status.hostIP
+        volumeMounts:
+        - name: otel-collector-config-vol
+          mountPath: /conf
+        livenessProbe:
+          httpGet:
+            path: /
+            port: 13133 # Health Check extension default port.
+        readinessProbe:
+          httpGet:
+            path: /
+            port: 13133 # Health Check extension default port.
+      volumes:
+        - configMap:
+            name: otel-collector-conf
+            items:
+              - key: otel-collector-config
+                path: otel-collector-config.yaml
+          name: otel-collector-config-vol
+EOF
+```
+This will create 2 configmaps `` and ``. `otel-agent` is created as a `DaemonSet` and `otel-collector` is created as a `Deployment`
+
+```
+watch oc get configmaps, ds, deployment, services
+```
+Once the service is up and running take the `ClusterIp` and update the otlp exporter endpoint in the configmap as
+
+```
+ oc edit cm/otel-agent-conf -o yaml
+
+ exporters:
+      logging:
+      otlp:
+        endpoint: "ClusterIP:4317" # replace with the ClusterIP for otel-collector service
+
+```
+Now delete the three agent pods so that the otel-agent DaemonSet can launch new pods with updated endpoint. 
+
+`oc delete pods --selector=component=otel-agent`
+
+Check otel-collector pod logs to see traces. You should see traces like
+```
+2021-06-15T13:38:56.990Z        INFO    loggingexporter/logging_exporter.go:42  TracesExporter  {"#spans": 110}
+2021-06-15T13:38:58.995Z        INFO    loggingexporter/logging_exporter.go:42  TracesExporter  {"#spans": 23}
+2021-06-15T13:39:02.001Z        INFO    loggingexporter/logging_exporter.go:42  TracesExporter  {"#spans": 55}
+2021-06-15T13:39:04.005Z        INFO    loggingexporter/logging_exporter.go:42  TracesExporter  {"#spans": 77}
+```
+
+Since `Jaeger` is not running right now you will also notice below error in the collector log but that will be resolved as soon as you install and create Jaeger. 
```

Comment:
```suggestion
Since `Jaeger` is not running right now, you will also notice the below error in the collector log, but that will be resolved as soon as you install and create Jaeger. 
```